services/common/snmpwalk_helpers.py

```diff
@@ -0,0 +1,111 @@
+# Check for which J2735 PSID matches val
+# BSM  - 20
+# SPaT - 8002
+# TIM  - 8003
+# MAP  - E0000017
+# SSM  - E0000015
+# SRM  - E0000016
+def message_type_rsu41(val):
+    # Hex octets are spaced out in the output and are always 4 octets long
+    if val == '" "' or val == "00 00 00 20" or val == "00 00 00 32":

```

Comment:
What are these "extra" compares looking at in here? The `00 00 00 32` for BSM, `00 03 27 70` for SPaT, etc? Maybe we should add a comment in here clarifying what were checking since the above comment seems to differ.

Reply:
I can update the comment, these are various formats we have encountered that we have needed to implement to support RSUs. This could be streamlined if we could use the RSU-MIB on the snmpwalk but the PSID limitation prevents us from doing this.

Reply:
Are these RSU manufacturer specific values? I don't believe they translate out to the standard octet values we're expecting so they seem strange. Let's do update the comment here to clarify.

Reply:
There isn't a great way to explain it besides it is the many values I have encountered. These can change from firmware version to firmware version on the same RSU type.

Reply:
I added more to the comment to explain it isn't an exact science lol